packages/terra-notification-dialog/src/NotificationDialog.module.scss

```
@@ -24,15 +24,15 @@
     flex-direction: row;
     height: 100%;
     overflow: auto;
-    padding: var(--terra-notification-dialog-body-padding, 1.25rem);
+    padding: var(--terra-notification-dialog-body-padding, 0.71428rem 0.85714rem 1rem 0.92857rem);
```

design change - update to bottom padding:
```suggestion
    padding: var(--terra-notification-dialog-body-padding, 0.71428rem 0.85714rem 1.42857rem 0.92857rem);
```

---

Updated here 23d23edd40f5e5ce50c864566fdd998fdee95882